library/Garp/Cli/Command/Email.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Garp_Cli_Command_Email
+ * Send email using the CLI
+ *
+ * @package Garp_Cli_Command
+ * @author  Floris S. Koch <floris@grrr.nl>
+ */
+class Garp_Cli_Command_Email extends Garp_Cli_Command_Flow {
+    /**
+     * Send email
+     *
+     * @return void
+     */
+    public function send() {
+        $mailer = new Garp_Mailer;
+
+        if (getenv('CLI_EMAIL_SENDER')) {
+            $mailer->setFromAddress(getenv('CLI_EMAIL_SENDER'));
+        }
+
+        Garp_Cli::lineOut('Please fill the following fields');
+        $email = array(
+            'to' => Garp_Cli::prompt('Recipient:'),
+            'subject' => Garp_Cli::prompt('Subject:'),
+            'message' => Garp_Cli::prompt('Body text:'),
+        );
```

Comment:
Ik zou het niet via `prompt` doen eigenlijk, dan is het een interactief proces en moeilijk te automatiseren.   
Als je de parameters kan opgeven als argumenten van de commandline zijn we wat flexibeler.

Reply:
Oke, ja maakt mij niet uit. Het primaire doel is om de SES settings voor spambeveiliging in praktijk te kunnen testen. Hoe gaan we dat testen?

Reply:
Je hebt services waar je een mail naartoe kunt sturen.   
Maar met `prompt` maak je het commando per definitie interactief, het kan dus alleen door een human worden uitgevoerd, en niet als onderdeel van Travis ofzo.  
Ik vind het sterker (ook omdat het maar 3 parameters zijn) om ze als commandline arguments te gebruiken.